MinMaxTypeValues/MinMaxTypeValues/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+
+namespace MinMaxTypeValues
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string byteType = "BYTE";
```

Comment:
Also remove string variables.

Reply:
String variables of data types are removed.